client/blocks/reader-full-post/_content.scss

```diff
@@ -8,15 +8,26 @@
 	overflow-wrap: break-word;
 	word-wrap: break-word;
 
-	h1 {
-		font-size: 28px;
+	h1,
+	h2,
+	h3,
+	h4,
+	h5,
+	h6 {
 		font-weight: 700;
+		margin-bottom: 15px;
+		margin-top: 30px;
+		line-height: 1.1;
+		text-align: left;
+	}
+
+	h1 {
+		font-size: 54px;
 		margin: 0 0 16px;
 	}
 
 	h2 {
-		font-size: 24px;
-		font-weight: 700;
+		font-size: 47px;
```

Comment:
This is close to but does not match: https://github.com/Automattic/wp-calypso/blob/a83f9acf2b30fc352da61669616f280df655134b/packages/typography/styles/variables.scss#L37